bladeui/js/pages/referralsForm.js

```diff
@@ -0,0 +1,90 @@
+"use strict";
+
+const BaseClass = require("./baseClass");
+
+class ReferralsForm extends BaseClass
+{
+  constructor(props)
+  {
+    super(props);
+  }
+
+  initListeners()
+  {
+    this.backButton = document.getElementById("back-button");
+    this.sendButton = document.getElementById("send-referral-btn");
+    this.email = document.getElementById("email");
+    this.message = document.getElementById("message");
+
+    this.backButton.addEventListener("click",
+      this.handleOpenPreviousView.bind(this));
+    this.sendButton.addEventListener("click",
+      this.handleSubmitReferralForm.bind(this));
+    this.email.addEventListener("change", this.onChangeEmail.bind(this));
+    this.message.addEventListener("change", this.onChangeMessage.bind(this));
+  }
+
+  handleOpenPreviousView()
+  {
+    super.handleChangeView("referralsMenuView");
+  }
+
+  onChangeEmail(e)
+  {
+    if (this.validateEmail(e.target.value))
+    {
+      this.email.classList.remove("input-invalid");
+    }
+  }
+
+  onChangeMessage(e)
+  {
+    if (e.target.value)
+    {
+      this.message.classList.remove("input-invalid");
+    }
+  }
+
+  validateEmail(email)
+  {
+    const re = /\S+@\S+\.\S+/;
```

Comment:
Please, change the regex variable name to express it's purpose. I.E. `emailRegex`